components/browser/errorpages/src/main/assets/error_style.css

```diff
@@ -32,6 +32,7 @@ ul {
   padding: 0 1em;
   margin: 0;
   list-style: round outside none;
+  role: presentation;
```

Comment:
I don't think this is a valid CSS property. Maybe you meant to use [the HTML attribute](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/presentation/PresentationRoleExamples.html)?